### PR TITLE
Arshad ariff patch 2 updated requested changes finalized!

### PR DESCRIPTION
**Pull request by ArshadAriff**
Updated data-type.md

### CODE REVIEW

File: docs/docs/scheme-intro/data-types.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Escape character used by Terminal emulators to add formatting (like colors).
 
 ## Characters
 
-You can define single character as data type
+Single characters can be defined as a distinct data type.
```
Comment (jcubic) [inline docs/docs/scheme-intro/data-types.md:62]: I prefer:

> You can define a single character as a data type

I don't see a reason to change the structure of the sentence and swap stuff inside.

**Review by jcubic — CHANGES_REQUESTED** [review]
You don't need to Open new PR with the fix. Just fix the issue with next commit. I will merge the PR if you fix this.

**Comment by jcubic** [comment]
Also, it would be nice if you write in the PR what you're doing.

> Arshad ariff patch 2

and

> Updated data-type.md

Says nothing what the change is about.

**Comment by ArshadAriff** [comment]
Done! @jcubic

**Review by ArshadAriff — COMMENTED** [review]
Done with the requested changes

**Pull request merged**